src/old/lib/utilities/API/api.ts

```diff
@@ -166,6 +166,9 @@ export const setFakePostViewsCounter = async (
       views,
     },
   });
+  // const result = instance.post(`/post/set-fake-view/${id}?views=${views}`);
```

Comment:
is it needed? :)

Reply:
Most likely no. In any case, unfortunately, there is quite a lot of this. it probably even needs to be highlighted in a separate issue, but now the task is to upload the changes from develop to release :)